consumers/notify/api/requests.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"github.com/mainflux/mainflux/consumers/notify"
+	"github.com/mainflux/mainflux/pkg/errors"
+)
+
+var (
+	errInvalidTopic   = errors.New("invalid Subscritpion topic")
+	errInvalidContact = errors.New("invalid Subscritpion contact")
+	errNotFound       = errors.New("invalid or empty Subscription id")
+)
+
+type createSubReq struct {
+	token   string
+	Topic   string
```

Comment:
Is there a reason to omit unmarshal string?

Reply:
Yes, `topic` is extracted from the URL path, rather than the JSON body.

Reply:
Should it be exported (uppercase) in that case?